src/backend/Controllers/OffersController.cs

```diff
@@ -23,7 +24,7 @@ public Offer Get(int id)
             // should return an offer having a specified id
             // the id will likely be index in a static list
             // until a database is created
-            return new Offer();
+            return OffersServices.GetSpecificOffer(id);

```

Comment:
Would be better if called `GetOfferById`
```suggestion
            return OffersServices.GetOfferById(id);

```